test/fixtures/common-cases.js

```diff
@@ -108,6 +113,31 @@ module.exports = {
   'containsSymlink/': {
     code: 404,
     body: '<h1>404</h1>\n'
+  },
+
+  'gzip/real_ecstatic': {
```

Comment:
Can we do some asserts for content-type headers?

Note that if any tests that used to pass pre-change and start to fail post-change are potentially breaking, and need to be discussed, so you should confirm how they behave prior to your PR.